config/brakeman.ignore

```diff
@@ -1,39 +1,5 @@
 {
   "ignored_warnings": [
-    {
```

Comment:
During the update of Brakeman ignore file, I went through deprecated alerts that weren't relevant anymore and removed them from the ignore file. Hence these blocks of removed code.